Documentation: Add meta data. (#486)
plone/plone.api: docs/group.md

---
html_meta:

>   "description": "Create, access, modify and delete groups and group memberships"

Reviewer: ```suggestion
  "description": "Create, access, modify, and delete groups and group memberships"
```

>   "property=og:description": "Create, access, modify and delete groups and group memberships"

Reviewer: ```suggestion
  "property=og:description": "Create, access, modify, and delete groups and group memberships"
```

## Create group

> Groups are created with {meth}`api.group.create` providing a group id.

Reviewer: ```suggestion
To create a new group, use {meth}`api.group.create`, providing a required string parameter of `groupname`.
```

```{todo}
Why is the api parameter for the id named 'groupname'?

> ```

Reviewer: Wild guess: "ID" is usually a positive integer, whereas "name" usually is a string. Hence "groupname" instead of "groupid".

## Editing a group

> Groups can be edited by using the tool `portal_groups`.

Reviewer: ```suggestion
Groups can be edited by using the method {meth}`api.portal.get_tool`, providing the portal tool `portal_groups` as the `name` argument.
```

> In this example, the `title`, `description` and `roles` are updated for the group 'Staff'.

Reviewer: ```suggestion
In this example, the `title`, `description` and `roles` are updated for the group `staff`.
```
